src/iosurface.cs

```diff
@@ -0,0 +1,240 @@
+//
+// IOSurface bindings
+//
+// Authors:
+//	Miguel de Icaza  <miguel@microsoft.com>
+//
+// Copyright 2017 Microsoft Inc. All rights reserved.
+//
+
+#if XAMCORE_2_0
+
+using System;
+using XamCore.CoreFoundation;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.IOSurface {
+
+	[Static]
+	[Internal]
+	interface IOSurfacePropertyKey {
+		[Field ("IOSurfacePropertyAllocSizeKey")]
+		[iOS (10,0), Mac (10,12)]
```

Comment:
You can apply this directly to the interface instead of applying to all members, this will reduce generated metadata